isis/src/control/objs/BundleSettings/BundleSettings.cpp

```diff
@@ -217,6 +219,30 @@ namespace Isis {
   }
 
 
+      /**
+       * @brief BundleSettings::setCubeList
+       * @param cubeList
+       *
+       */
+  void BundleSettings::setCubeList(QString cubeList)    {
+
+        m_cubeList = cubeList;
+
+
```

Comment:
Why all the white space here?

Reply:
@Kelvinrr I have removed the white space.